creativecommons/vocabulary
# [Fix] Adds padding to content in default-page.html

## Fixes



- Fixes #68 by @SisiVero

## Description

This pull request fixes the responsiveness issues of the content area of `default-page.html`

## Technical details

- Adds padding, defined by the `--vocabulary-page-edges-space` variable, to the `.content` class at screen sizes `680px` or less

## Tests

1. Open the `specimen/contexts/default-page.html` file in your browser of choice
2. Using chrome developer tools, toggle responsive mode and reduce screen size below 680px
3. Observe that there is adequate padding for content on the page.

## Screenshots

![image](https://github.com/user-attachments/assets/3d97b011-03e6-4b44-a02e-8174539c8261)


## Checklist



  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin




For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
